src/gtp/mod.rs

```diff
@@ -78,6 +78,7 @@ impl<'a> GTPInterpreter<'a> {
             "showboard",
             "time_left",
             "time_settings",
+            "uct_gfx",
```

Comment:
According to the GTP spec these engine specific command should start with a prefix. That's where all these `kgs-*` commands come from. Would it be possible to rename it to `imrscl-uct_gfx`?